src/h5web/vis-packs/core/visualizations.test.ts

```diff
@@ -8,16 +8,24 @@ import {
   makeSimpleShape,
   makeDataset,
   makeSimpleDataset,
+  booleanType,
 } from '../../providers/mock/metadata-utils';
 
 const datasetIntScalar = makeDataset('dataset_int', scalarShape, intType);
 const datasetFltScalar = makeDataset('dataset_flt', scalarShape, floatType);
 const datasetStrScalar = makeDataset('dataset_flt', scalarShape, stringType);
+const datasetBoolScalar = makeDataset('dataset_bool', scalarShape, booleanType);
 const datasetInt0D = makeSimpleDataset('dataset_int_0d', intType, []);
 const datasetInt1D = makeSimpleDataset('dataset_int_1d', intType, [5]);
 const datasetInt2D = makeSimpleDataset('dataset_int_2d', intType, [5, 3]);
 const datasetStr2D = makeSimpleDataset('dataset_str_2d', stringType, [5, 3]);
 const datasetFlt3D = makeSimpleDataset('dataset_flt_3d', intType, [5, 3, 1]);
+const datasetBool4D = makeSimpleDataset('dataset_flt_3d', booleanType, [
+  2,
+  3,
+  3,
+  1,
+]);
```

Comment:
Does this have to be a 4D array ? 1D would suffice and maybe fit on one line 😂  (also the name of the dataset is incorrect).